content/https/http-3/contents.lr

```diff
@@ -0,0 +1,14 @@
+_model: question
+---
+title: What's the difference between Tor and DoH (DNS over HTTPS)?
+---
+seo_slug: difference-between-Tor-DoH
+---
+description:
+DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol. A goal of the method is to increase user privacy and security by preventing eavesdropping and manipulation of DNS data by man-in-the-middle attacks by using the HTTPS protocol to encrypt the data between the DoH client and the DoH-base DNS resolver. Encryption by itself does not protect privacy, encryption is simply a method to obfuscate the data.

```

Comment:
Please try to add some linebreaks to this long lines, to make translation easier.
For example this line could be:
DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol.
A goal of the method is to increase user privacy and security by preventing eavesdropping and manipulation of DNS data by man-in-the-middle attacks by using the HTTPS protocol to encrypt the data between the DoH client and the DoH-base DNS resolver.
Encryption by itself does not protect privacy, encryption is simply a method to obfuscate the data.